src/main/java/com/gd/clinic/service/PatientService.java

```diff
@@ -0,0 +1,62 @@
+package com.gd.clinic.service;
+
+import com.gd.clinic.entity.Patient;
+import com.gd.clinic.repository.PatientRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+// TODO: Uncomment once security has been merged to this branch or vice versa
+//import org.springframework.security.core.context.SecurityContextHolder;
+import javax.transaction.Transactional;
+import java.time.OffsetDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class PatientService {
+
+    private final PatientRepository patientRepository;
+
+    public Patient getById(UUID id) throws RuntimeException {
+        Optional<Patient> fetchedPatient = patientRepository.findById(id);
```

Comment:
please use here `findById(id).orElseThrow(() -> YourException.class)`
and better to create custom exception for this case, for example `EntityNotFoundException`

Reply:
Done, changed this method to `return patientRepository.findById(id).orElseThrow(() -> new RuntimeException());` this commit, much shorter.